contracts/Shortfall/Shortfall.sol

```diff
@@ -26,7 +28,13 @@ import { EXP_SCALE } from "../lib/constants.sol";
  * if the risk fund covers the pool's bad debt plus the 10% incentive, then the auction winner is determined by who will take the smallest percentage of the
  * risk fund in exchange for paying off all the pool's bad debt.
  */
-contract Shortfall is Ownable2StepUpgradeable, AccessControlledV8, ReentrancyGuardUpgradeable, TokenDebtTracker {
+contract Shortfall is
+    TimeManagerV8,
```

Comment:
Shortfall itself defines storage variables too, so due to gap variable there will be storage collision I think.
We will have to remove gap variable to avoid it.

Reply:
Or we can make a storage contract for the shortfall states which would be inherited in the shortfall contract to prevent storage collision and then inherit the TimeManager, in this way there is no need to remove the gap from TimeManager.

Reply:
Similar to the RewardsDistributor scenario, I would go with the second option

Reply:
Done https://github.com/VenusProtocol/isolated-pools/pull/324/commits/d6311f25cf84ac1c82a143109e45ad9fea1d4dc3
These are the snippets of storage verifying that storage layout is correct.
![Screenshot from 2024-01-12 15-01-07](https://github.com/VenusProtocol/isolated-pools/assets/104391977/f9333df5-5ce2-4e0f-be31-087d2f18659a)
![Screenshot from 2024-01-12 15-00-48](https://github.com/VenusProtocol/isolated-pools/assets/104391977/33170554-e34b-4bb5-b65b-05d7daa33ddf)